src/ship_commands.cpp

```diff
@@ -8,15 +8,6 @@
 
 
 
-static inline float fmin( float a, float b )
```

Comment:
Same here, you have to include them from <math.h>

Reply:
`ship_commands.cpp` includes `"cmd/unit_generic.h"`. So it will automatically have `<math.h>` available, via much the same dependency chain as `unit_generic.cpp`.

Reply:
I have built and run this entire project on Linux. Everything works, more or less. I want to build Vega Strike on Windows as well, but I'm running into some difficulties on that front, configuring CMake.

Reply:
`<math.h` included explicitly, as requested.

Reply:
Thanks